wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclReader.java

```diff
@@ -0,0 +1,437 @@
+// Generated from ecl.g4 by ANTLR 4.5
```

Comment:
Removed that line, the EclReader wasn't generated; I started creating it based on the generated BaseReader and never removed that line